scania-digital-design-system/sdds
# Feature/colour package

**Describe pull-request**  
- Add colour palette
- Remove old bootstrap colour scheme variables from global root
- Add new value to all components and make it available in :root

**Solving issue**  

Fixes: https://github.com/scania/corporate-ui/issues/592

**How to test**  
1. Open Storybook > Colour
2. Inspect element >> check in :root, html css vars for colours are correct
3. Check if all colour is available
